runner/types.go

```diff
@@ -52,7 +52,7 @@ type Result struct {
 	FavIconMMH3        string                 `json:"favicon,omitempty" csv:"favicon"`
 	FaviconPath        string                 `json:"favicon_path,omitempty" csv:"favicon_path"`
 	FinalURL           string                 `json:"final_url,omitempty" csv:"final_url"`
-	ResponseHeader     map[string]interface{} `json:"header,omitempty" csv:"header"`
+	ResponseHeaders    map[string]interface{} `json:"headers,omitempty" csv:"headers"`
 	RawHeader          string                 `json:"raw_header,omitempty" csv:"raw_header"`
```

Comment:
raw_headers*

Reply:
Sorry, I'm not sure to understand what you expect here ? :)